src/types/receiver.ts

```diff
@@ -1,23 +0,0 @@
-import { AckFn } from '../types';
```

Comment:
This file should be restored as it wouldn't be moved to the new external package